### ISSUE TITLE
List upper-roman style does not work with Google Chrome

### ISSUE THREAD

**Issue by doompadee** [tailwindlabs/tailwindcss-typography#192]
### What version of @tailwindcss/typography are you using?

0.4.1

### What version of Node.js are you using?

14.16.0

### What browser are you using?

Chrome 92.0.4515.131

### What operating system are you using?

Linux Fedora 33

### Reproduction repository

https://github.com/doompadee/typography

### Describe your issue

Ordered list style "upper-roman" does not work when using Google Chrome. It uses "lower-roman" instead.

Looks like a Chrome bug, it works fine with Firefox, but with the way Typography implements list bullets it does not seem possible to easily workaround the issue with client code, therefore a fix would be very welcome.

See https://doompadee.github.io/typography/upper-roman-list.html

**Comment by christoferd** [comment]
I cannot find "upper-roman" in the documentation.

Found: **font-serif uppercase**

[https://tailwindcss.com/docs/font-family](url)
[https://tailwindcss.com/docs/text-transform#uppercase](url)

**Comment by thien-do** [comment]
Tailwind uses the "type" attribute for that, maybe this will help:

![image](https://user-images.githubusercontent.com/5953369/133922801-776cc87f-16a6-4133-a7c3-2e76b8348e73.png)

So far I see `a` `A` `i` and `I`

**Comment by evanfuture** [comment]
This also throws a warning during the build:
```
⠋ Generating index html...4 rules skipped due to selector errors:
  .prose ol[type=A s] -> Malformed attribute selector: type=A s]
  .prose ol[type=a s] -> Malformed attribute selector: type=a s]
  .prose ol[type=I s] -> Malformed attribute selector: type=I s]
  .prose ol[type=i s] -> Malformed attribute selector: type=i s]
```

**Comment by simonswiss** [comment]
Hey!

As this Stackoverflow answer explains quite well, Firefox is the only browser that implements the experimental case-sensitive comparison (the `s` in `ol[type="A"s]` – that the past two comments mention).

---

![image](https://user-images.githubusercontent.com/485747/155251293-9bac5cbb-a700-4443-af10-0da30cccca6d.png)

---

Without this, both `A` and `a` types (and same for `I` and `i`) will be matched. Since the generated styles in the Typography plugin output the `type="a"` selector _after_ the `type="A"`, the "lower" type styles are applied in Chrome.

---

![image](https://user-images.githubusercontent.com/485747/155251404-a94dae1d-9876-405d-8e6d-388c6607d121.png)

---

It works in Firefox, because the case-sensitive check is respected, and the override doesn't happen.

A workaround here (which will only work if you want all your `li` markers uppercase): add a `marker:uppercase` class alongside your `prose` class in your markup:

---

![image](https://user-images.githubusercontent.com/485747/155251107-4f9e8595-9d87-4884-a142-99ed8a44a427.png)

---

To sum it up, this is not really a bug with Chrome, but rather an experimental feature that is only available in Firefox.
